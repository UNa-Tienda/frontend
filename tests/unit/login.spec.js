import Login from "../../src/views/Iniciar_sesion.vue"
import {mount} from "@vue/test-utils"

describe("mount component Login",()=>{
    it("it renders the component",()=>{
        const wrapper = mount(Login)
        expect(wrapper.html()).toMatchSnapshot()
    })
})
