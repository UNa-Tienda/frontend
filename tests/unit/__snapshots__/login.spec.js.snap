// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount component Login it renders the component 1`] = `
<div class="container-fluid containerExternoIS">
  <div class="row">
    <div class="col-12"><br>
      <router-link to="[object Object]"><img src="../assets/logo.png" alt="Logo" class="logoregister"></router-link>
      <div id="login" class="login">
        <b-form class="datosIS col-xs-12 col-sm-8 col-md-6 col-lg-4 border border-primary rounded-lg">
          <h2 class="col-12 text-center text-primary mt-3 mb-3">Login</h2> <img src="../assets/IS.png" alt="ISL" class="IS">
          <b-form-group id="input-group-1" label="Nickname" class="form-group col-12 mb-4">
            <b-form-input id="input-1" type="username" required="" placeholder="Ingrese su nombre de usuario"></b-form-input>
          </b-form-group>
          <b-form-group id="input-group-2" label="Contraseña" class="form-group col-12 mb-4">
            <b-form-input id="input-2" required="" placeholder="Ingrese su contraseña" type="password"></b-form-input>
          </b-form-group>
          <div align-v="center" class="text-center">
            <b-button size="lg" type="sumbit" variant="primary">Iniciar Sesión</b-button>
          </div> <br>
          <div align-v="center" class="text-center">
            <router-link to="[object Object]">¿Olvidaste tu contraseña?</router-link>
          </div> <br>
          <div align-v="center" class="text-center">
            <b-button size="lg" to="register" variant="secondary">Registrarse</b-button>
          </div> <br>
        </b-form>
      </div>
    </div>
  </div>
</div>
`;
